pandas/io/formats/style.py

```diff
@@ -901,16 +901,21 @@ def background_gradient(self, cmap='PuBu', low=0, high=0, axis=0,
     def _background_gradient(s, cmap='PuBu', low=0, high=0):
         """Color background in a range according to the data."""
         with _mpl(Styler.background_gradient) as (plt, colors):
-            rng = s.max() - s.min()
+            min = s.values.min()
+            max = s.values.max()
```

Comment:
Can you use different names for `min`/`max` so they don't override the built-in `min`/`max` functions?

Reply:
sure, updated commit